vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -614,6 +614,11 @@ void WindowsTextInputComponentView::updateProps(
     propBits |= TXTBIT_CHARFORMATCHANGE;
   }
 
+  if (oldTextInputProps.secureTextEntry != newTextInputProps.secureTextEntry) {
+    propBitsMask |= TXTBIT_USEPASSWORD;
```

Comment:
This will still set TXTBIT_USEPASSWORD if you have a textinput and set then unset the secureTextEntry property

Reply:
You should set the TXTBIT_USEPASSWORD when
`oldTextInputProps.secureTextEntry != newTextInputProps.secureTextEntry`, 
then only set TXTBIT_USEPASSWORD when 
`newTextInputProps.secureTextEntry == true`